### PR TITLE
T4866: rewrite show_interfaces.py show* functions to standardized op-mode

### PR DESCRIPTION
**Pull request by jestabro**
## Change Summary

Rewrite the collection of 'show' functions in op-mode 'show_interfaces.py' to standardized form using vyos.opmode. Functions involving clear/reset of interface counters will be handled in sub-task [T4885](https://phabricator.vyos.net/T4885).

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [ ] Migration from an old Vyatta component to vyos-1x, please link to related PR inside obsoleted component
- [ ] Other (please describe):

## Related Task(s)

* https://phabricator.vyos.net/T4866

## Checklist:

- [X] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-1x/blob/current/CONTRIBUTING.md) document
- [X] I have linked this PR to one or more Phabricator Task(s)
- [ ] I have run the components [**SMOKETESTS**](https://github.com/vyos/vyos-1x/tree/current/smoketest/scripts/cli) if applicable
- [X] My commit headlines contain a valid Task id
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: src/op_mode/interfaces.py [hunk 1]
```diff
@@ -0,0 +1,406 @@
...
+            continue
+
+        if vrrp:
+            vrrp_interfaces = VRRP.active_interfaces()
+            if ifname not in vrrp_interfaces:
+                continue
+
+        yield interface
+
+def _split_text(text, used=0):
```
Comment (sarthurdev) [inline src/op_mode/interfaces.py:85]: Do you think this should be moved into `vyos.util`?

I believe the warning classes also has code to fit into a screen: https://github.com/vyos/vyos-1x/blob/current/python/vyos/base.py

**Review by sarthurdev — APPROVED** [review]

File: src/op_mode/interfaces.py (hunk 1 above)
Comment (jestabro) [reply src/op_mode/interfaces.py:85]: I agree; this code is from the original implementation and should be replaced with a general tool. I'll address as a separate task.

**Comment by jestabro** [comment]
Force push to fix spelling and return value.

**Review by sever-sever — APPROVED** [review]
Internal tests that I did looks good
Approve

File: src/op_mode/interfaces.py [hunk 2]
```diff
@@ -0,0 +1,412 @@
...
+from datetime import datetime
+from tabulate import tabulate
+
+import vyos.opmode
+from vyos.ifconfig import Section
+from vyos.ifconfig import Interface
+from vyos.ifconfig import VRRP
+from vyos.util import cmd, rc_cmd, call
+
+def catch_broken_pipe(func):
```
Comment (dmbaturin) [inline src/op_mode/interfaces.py:33]: I wonder if  this deserves a promotion to `vyos.util` even.

**Review by dmbaturin — APPROVED** [review]

**Pull request merged**